controller_manager/src/controller_manager.cpp

```diff
@@ -618,6 +709,14 @@ void ControllerManager::start_controllers_asap()
 #endif
 }
 
+void ControllerManager::list_controller_interfaces_srv_cb(
```

Comment:
```suggestion
void ControllerManager::list_resource_interfaces_srv_cb(
```

Reply:
We should also return the interfaces claimed status here. Very useful for introspection.

Reply:
ec56888